newrelic/api/ml_model.py

```diff
@@ -52,3 +53,34 @@ def get_ai_message_ids(response_id=None):
         return [{"conversation_id": conversation_id, "request_id": request_id, "message_id": _id} for _id in ids]
     warnings.warn("No message ids found. get_ai_message_ids must be called within the scope of a transaction.")
     return []
+
+
+def record_ai_feedback(
```

Comment:
```suggestion
def record_ai_feedback_event(
```

Reply:
I saw it both ways in the CDD but it makes sense to converge on `record_ai_feedback_event` for consistency with other API names :)